ckanext/ontario_theme/fanstatic/scripts/odc-access-level.js

```diff
@@ -29,4 +29,17 @@
       }
   }
 
+  window.addEventListener('load', updateAccessLevelSentence)
+
+  function updateAccessLevelSentence() {
+    var selectedButton = document.querySelector('input[name="access_level"]:checked');
+    if (selectedButton) {
+      var displayName = selectedButton.getAttribute('data-display-name').trim();
+      var display_name_without_count = displayName.replace(/\s*\(\d+\)$/, '');
+      var calloutElement = document.querySelector('.access-level-status');
+      calloutElement.querySelector('#access-level-sentence-value').textContent = display_name_without_count;
```

Comment:
```suggestion
      var displayName = selectedButton.getAttribute('data-display-name').trim();
      var calloutElement = document.querySelector('.access-level-status');
      calloutElement.querySelector('#access-level-sentence-value').textContent = displayName;
```

Reply:
Shouldn't I keep line 38 to get rid of the count value?

Reply:
![data-display-name](https://github.com/ongov/ckanext-ontario_theme/assets/60950452/9b84ee3f-d9c9-46e3-b994-9ae1e700af93)
In the HTML, the data-display-name attribute is just the name of the radio button with no count. So we won't need line 38